CHANGELOG.md

```diff
@@ -5,6 +5,19 @@
 - Add driver option --silent to disable logging
 -->
 
+
+## v1.5.1
+- Fix issue there're some applications that launched process exists immediately, so in close/quit function it throws exception Process Not Found.

```

Comment:
Need brief info just like other changelogs below (like this https://github.com/PersonifyInc/Winium.Desktop/pull/1/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR25 or this https://github.com/PersonifyInc/Winium.Desktop/pull/1/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR40) - consider if we want to ask for a merge to upstream at 2gis/Winium.Desktop, we need to be consistency and brief